src/pages/fi-pv-expenses/forms/PaymentVoucherExpensesForm.js

```diff
@@ -0,0 +1,739 @@
+      component: 'textfield',
+      label: labels.expenseName,
+      name: 'etName',
+      props: {
+        readOnly: true
+      }
+    },
+    {
+        component: 'textfield',
+        label: labels.subtotal,
+        name: 'subtotal',
+        props: {
+          readOnly: true
+        }
+    },
+    {
+        component: 'checkbox',
+        label: labels.isVAT,
+        name: 'isVAT',
+        async onChange({ row: { update, newRow } }) {
+          if (newRow.isVAT && newRow.amount) {
+            let newSubtotal = newRow.amount * (100 / (100 + formik.values.vatPct));
+            update({
+              subtotal: newSubtotal.toFixed(2),
+              vatAmount: (newRow.amount - newSubtotal).toFixed(2)
+            })
+          } else {
+            update({
+              subtotal: newRow.amount,
+              vatAmount: 0
+            })
+          }
+        }
+    },
+    {
+        component: 'textfield',
+        label: labels.vat,
+        name: 'vatAmount',
+        props: {
+          readOnly: true
+        }
+    },
+    {
+        component: 'numberfield',
+        label: labels.amount,
+        name: 'amount',
+        props: {
+          readOnly: false
+        },
+        async onChange({ row: { update, newRow } }) {
+          if (newRow.isVAT) {
+            let newSubtotal = newRow.amount * (100 / (100 + formik.values.vatPct));
+            update({
+              subtotal: newSubtotal.toFixed(2),
+              vatAmount: (newRow.amount - newSubtotal).toFixed(2)
+            })
+          } else {
+            update({
+              subtotal: newRow.amount,
+              vatAmount: 0
+            })
+          }
+        }
+    },
+    {
+        component: 'textfield',
+        label: labels.vendorName,
+        name: 'supplierName',
+        props: {
+          readOnly: false
+        }
+    },
+    {
+      component: 'textfield',
+      label: labels.vatNo,
+      name: 'taxRef',
+      props: {
+        readOnly: false
+      }
+  },
+  {
+      component: 'textfield',
+      label: labels.notes,
+      name: 'notes',
+      props: {
+        readOnly: false
+      }
+  },
+  {
+      component: 'button',
+      name: 'hasCostCenters',
+      img: 'costCenters',
+      label: labels.costCenter,
+      onClick: (e, row, update, updateRow) => {
+        stack({
+          Component: ExpensesCostCenters,
+          props: {
+            labels: labels,
+            recordId: recordId,
+            row,
+            updateRow
+          },
+          width: 700,
+          height: 600,
+          title: labels.costCenter
+        })
+      }
+    }
+  ]
+
+  const getCostCenters = async (pvId, seqNo) => {
+    const res = await getRequest({
+      extension: FinancialRepository.PaymentVoucherCostCenters.qry,
+      parameters: `_pvId=${pvId}&_seqNo=${seqNo}`
+    });
+  
+    return res.list.map(item => ({
+      ...item,
+      id: item.ccSeqNo,
+      seqNo: item.seqNo,
+      ccSeqNo: item.ccSeqNo,
```

Comment:
it's neccessary to add these??
   seqNo: item.seqNo,
      ccSeqNo: item.ccSeqNo,
      ccName: item.ccName,
      ccRef: item.ccRef,
      pvId: item.pvId,
      ccId: item.ccId,